categories/results.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz | Chess Blaze</title>
+
+    <link rel="stylesheet" href="https://blaze-ui.netlify.app/css/foundation.css" />
+    <link rel="stylesheet" href="https://blaze-ui.netlify.app/Components/navigation/navigation.css" />                      
+    <link rel="stylesheet" href="https://blaze-ui.netlify.app/components/button/button.css" />
+    <script src="https://kit.fontawesome.com/21124ac8ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/css/shared.css">
+    <link rel="stylesheet" href="/css/navbar.css">
+    <link rel="stylesheet" href="/categories/quiz.css">
+    <link rel="stylesheet" href="/categories/results.css">
```

Comment:
Just a suggestion you can import all of this in your index css.

Reply:
Noted! will implement asap.